enhancements/single-node-developer-cluster-profile.md

```diff
@@ -0,0 +1,514 @@
+---
+title: single-node-developer-cluster-profile
+authors:
+  - "@rkukura"
+reviewers:
+  - "@deads2k"
+  - "@LalatenduMohanty"
+  - "@wking"
```

Comment:
this needs at least all group leads to be aware of major changes.

Reply:
Is there a list of group leads somewhere?

Reply:
@mfojtik can you help?